libbeat/docs/queueconfig.asciidoc

```diff
@@ -76,3 +77,147 @@ will be immediately available for consumption.
 
 The default values is 0s.
 
+[float]
+[[configuration-internal-queue-spool]]
+=== Configure the file spool queue
+
+The file spooling queue stores all events in an on disk ring buffer. The spool
+has a write buffer, which new events are written to. Events written to the
+spool are forwarded to the outputs, only after the write buffer has been
+flushed successfully.
+
+The spool waits for the output to acknowledge or drop events. If the spool is
+full, no new events can be inserted. The spool will block. Space is freed only
+after a signal from the output has been received.
+
+On disk, the spool operates in pages. The `file.page_size` setting configures
+the files page size on creation time. The optimal page size depends on the
```

Comment:
"files page size" isn't grammatically correct. Do you mean:

file's page size?
file page size?
page size for files?

search for other instances of "files page size" and replace them with whichever is correct.

Reply:
`file's page size`. A file is divided into pages of `page_size`.

Note: as followup we want to add a FAQ entry on determining the optimal page size. These can differ by OS and filesystem. Tools and nomenclature is different on different OSes as well.